### DISCUSSION TITLE
Status error from device

### DISCUSSION THREAD

**Discussion by Nazarbayevv** [deconz-community/node-red-contrib-deconz#227] [Q&A]
Hello,

How can I receive status error from my devices? For example, when device is not connected. I can see this info near dconz input node, but I want to send me notification via messenger
![IMG_0366](https://github.com/deconz-community/node-red-contrib-deconz/assets/50654913/65082605-877c-460c-9c59-dca2846aeff6)

**Comment by Zehir** [comment]
Did you try with a catch node ?

**Comment by Nazarbayevv** [comment]
Do you mean event node?
This one:
![IMG_0367](https://github.com/deconz-community/node-red-contrib-deconz/assets/50654913/b06c402f-dae6-4d2e-8d70-5ece3a6bb608)

**Reply by Zehir** [reply]
No the node red one
See : https://nodered.org/docs/user-guide/handling-errors

**Reply by Nazarbayevv** [reply]
Oh, this one. Thanks. I will try.

**Comment by Zehir** [comment]
You can also check for the state.reachable boolean.
Note that in both case, it's take time for a device to be marked as unreachable

**Comment by Nazarbayevv** [comment]
Hello, I tried to use catch node. It didn’t show connecting error. But I use deconz in node together with Apple Homekit nod and I can see not reachable status:
![IMG_0574](https://github.com/deconz-community/node-red-contrib-deconz/assets/50654913/ea0d74d5-064a-485a-8df1-e69400ed44b0)
![IMG_0575](https://github.com/deconz-community/node-red-contrib-deconz/assets/50654913/688814d3-811c-4691-9355-84e6db244df9)
 
 I can take no_response status this way for every nod. Can I do it from single point(single nod)?
